docs/VSS2DDSIDL.md

```diff
@@ -79,19 +79,13 @@ IDL files can be supplied as input to one of the DDS implementation (e.g: Cycloe
 
 
 ```bash
-git clone https://github.com/eclipse-cyclonedds/cyclonedds.git
-pip3 install cyclonedds conan
-cd cyclonedds
-mkdir build
-cd build
-conan install .. --build missing
-cmake ..
-cmake --build . --target install
+pip3 install cyclonedds
```

Comment:
maybe the only thing we could improve is making this a dev dependency using `poetry add --group dev cyclonedds`?

Reply:
I have already added it to pyproject.toml (hopefully in the right way). So installing it manually is maybe not needed if you have built vss-tools with poetry? (But needed if you just download vss-tools from pypi, right?)